byron/crypto/src/Cardano/Crypto/Hashing.hs

```diff
@@ -92,7 +93,7 @@ import Cardano.Binary
 --   Made abstract in order to support different algorithms
 newtype AbstractHash algo a = AbstractHash SBS.ShortByteString
   deriving (Eq, Ord, Generic, NFData)
-  deriving NoUnexpectedThunks via UseIsNormalForm SBS.ShortByteString
+  deriving NoThunks via InspectHeap SBS.ShortByteString
```

Comment:
There is [a `NoThunks` instance for `ShortByteString`](https://github.com/input-output-hk/nothunks/blob/a4d072f973a6054518a1fc176608b7c5c8dc8917/src/NoThunks/Class.hs#L493), I would use that one instead (I know this is kind of orthogonal to this PR):
```haskell
deriving (Eq, Ord, Generic, NFData, NoThunks)
```